docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,199 +1,244 @@
 ---
-title: Getting Started with Desktop Access
-description: Passwordless access to Windows systems for local users
+title: Get started with Desktop Access
```

Comment:
```suggestion
title: Get Started with Desktop Access
```

If we're using sentence case in titles, we should edit the style guide and make a plan to standardize that

Reply:
I'd like to standardize on that. I'll update the style guide with that and some other stuff from the style guide I put together for previous projects.